.teamcity/hosted/utils/steps/DockerSteps.kt

```diff
@@ -0,0 +1,67 @@
+package hosted.utils.steps
+
+import hosted.utils.Utils
+import hosted.utils.models.ImageInfo
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildSteps
```

Comment:
Let's use the latest DSL version in our imports (it's not important for this PR, but it's better to add new Kotlin DSL without an exact version).

Reply:
@burnasheva I strongly agree with this point. We have a ticket for it, which will be implemented soon. We haven't included it for `2023.05.x` branch, as that'd require a few DSL (other than imports) and C# changes, which might make the branch unstable. It will be done within the main branch instead.
The repository will be moved to newer version of DSL packages, whose name won't explicitly include the version, thus will match TeamCity server, as per [Removing version from DSL API package names](https://www.jetbrains.com/help/teamcity/upgrading-dsl.html#packages-without-dsl-api-version).